### PR TITLE
Feat: Added Snackbar Component

### PR DESCRIPTION
**Pull request by shuntzhere**
Screenshot (26)

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
code looks clean to me

**Review by shubhamkumar648 — APPROVED** [review]
as i see  in your recent coomit you modifed all  ! good to merge

**Review by therajatg — APPROVED** [review]
Looks clean, you can go ahead and merge

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  margin: 0.1rem;
+  padding: 1rem 1.5rem;
+  background-color: var(--color-black);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-radius: 1%;
+}
+
+.fas {
```
Comment (Gautam2305) [inline components/snackbar/snackbar.css:25]: using this might give errors in future ...... but if you are sure that you won't use it...then it's fine ....

**Review by Gautam2305 — COMMENTED** [review]
everything else looks good....good for merge.

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (shuntzhere) [reply components/snackbar/snackbar.css:25]: It's a standard I defined. Will update it for future uses using a specific selector.

**Review by Gautam2305 — APPROVED** [review]

**Pull request merged**